docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -164,8 +164,10 @@ Bug fixes
   :py:func:`~pvlib.irradiance.aoi_projection` (:issue:`1185`, :pull:`1191`)
 * Update GFS product names for GFS v16. (:issue:`1202`, :pull:`1203`)
 * Take into account ``EgRef``, ``dEgdT``, ``irrad_ref`` and ``temp_ref`` when
-  calling :py:func:`~pvlib.pvsystem.calcparams_cec`.
-  (:issue:`1215`, :pull:`1216`)
+  calling :py:func:`~pvlib.pvsystem.calcparams_cec` from 
+  :py:meth:`~pvlib.pvsystem.PVSystem.calcparams_cec`. (:issue:`1215`, :pull:`1216`)
```

Comment:
It now says:

```rst
[...] when calling from :py:meth:`~pvlib.pvsystem.PVSystem.calcparams_cec`
```

As if it was necessary to call the method to reproduce the issue.

However, this issue affected the function implementation (`pvlib.pvsystem.calcparams_cec`). So calling it not from the method, but directly, was a way to reproduce the issue.

So is this addition required?

Reply:
I see I have confused things. I'll undo that.